examples/OpenGLWindow/GLInstancingRenderer.cpp

```diff
@@ -27,6 +27,8 @@ float shadowMapWorldSize=10;
 
 #include "OpenGLInclude.h"
 #include "../CommonInterfaces/CommonWindowInterface.h"
+#include "../SharedMemory/SharedMemoryPublic.h"//for b3VisualShapeData
+
```

Comment:
This is not a good dependency. OpenGLWindow should compile without using any SharedMemory file.